custom/enikshay/nikshay_datamigration/tests/test_create_enikshay_cases.py

```diff
@@ -64,19 +63,30 @@ def setUp(self):
         self.domain = Domain(name='enikshay-test-domain')
         self.domain.save()
 
-        loc_type = LocationType.objects.create(
-            code='phi',
-            domain=self.domain.name,
-            name='PHI',
-        )
+        locations = setup_enikshay_locations(self.domain.name)
+        self.sto = locations['STO']
```

Comment:
I'm gonna pull this out into a utility so that @mkangia and I can use it in the API